frontend/packages/dev-console/src/components/import/app/AppSection.tsx

```diff
@@ -47,6 +48,7 @@ const AppSection: React.FC<AppSectionProps> = ({ project, noProjectsAvailable })
         name="name"
         label="Name"
         helpText="A unique name given to the component that will be used to name associated resources."
+        isDisabled={formType.value && formType.value === 'edit'}
         required
```

Comment:
We should remove the required asterix if the field is disabled because it's not like the user can do anything about it.